### PR TITLE
PYIC-4571: Archive vcs boilerplate

### PR DESCRIPTION
**Pull request by lwebb-iig**
Adds boilerplate for the archive vcs lambda

### What changed

- Template and empty lambda added 

### Why did it change

- Pre-requisite for https://govukverify.atlassian.net/browse/PYIC-4555

### Issue tracking

- [PYIC-4571](https://govukverify.atlassian.net/browse/PYIC-4571)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-4571]: https://govukverify.atlassian.net/browse/PYIC-4571?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lambdas/archive-vcs/src/main/resources/IpvLambdaJsonLayout.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
```
Comment (MikeCollingwood) [inline lambdas/archive-vcs/src/main/resources/IpvLambdaJsonLayout.json:1]: Is this identical for the other lambda replaying cimit ci's?

File: lambdas/archive-vcs/src/test/java/uk/gov/di/ipv/core/archivevcs/ArchiveVcsHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.di.ipv.core.library.service.ConfigService;
+import uk.gov.di.ipv.core.library.verifiablecredential.service.VerifiableCredentialService;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@ExtendWith(MockitoExtension.class)
+class ReplayCimitVcsHandlerTest {
```
Comment (MikeCollingwood) [inline lambdas/archive-vcs/src/test/java/uk/gov/di/ipv/core/archivevcs/ArchiveVcsHandlerTest.java]:
```suggestion
class ArchiveVcsHandlerTest {
```

**Review by MikeCollingwood — DISMISSED** [review]
Think this can be merged after a small change

File: lambdas/archive-vcs/src/main/resources/IpvLambdaJsonLayout.json (hunk 1 above)
Comment (lwebb-iig) [reply lambdas/archive-vcs/src/main/resources/IpvLambdaJsonLayout.json:1]: @MikeCollingwood I believe these are identical/very similar for all of our lambdas and are put alongside the lambda code for the purposes of bundling the lambda artifacts for AWS

**Review by Joe-Edwards-GDS — APPROVED** [review]
All seems reasonable so far, although idk if there's any value in merging until you've got a bit more meat to it.

A couple of thoughts:
- Is 'ArchiveVcs' the right name? I think `RevokeVcs` might reflect the purpose better - but it's not the end of the world if it's a pain to change.
- Should we have a readme, so future devs know what this is for? (In general, rather than this specific case)

**Comment by lwebb-iig** [comment]
> * Is 'ArchiveVcs' the right name? I think `RevokeVcs` might reflect the purpose better - but it's not the end of the world if it's a pain to change.

Revoke VCs makes sense - I think I went with archive as we had the concept of the new "archive" table but revoke probably makes more sense (and obvious that we're deleting things from the main table). Easy to change in a future PR (cloud-formation should gracefully handle the update).

> * Should we have a readme, so future devs know what this is for? (In general, rather than this specific case)

Yep, good idea - I think this was more of a starting point / branch that @MikeCollingwood & I could work from, but I'll bring this forward & add this to the list of tasks we came up with for this work.

**Pull request merged**